docs/activities/tf-scripting/index.html

```diff
@@ -5,7 +5,37 @@
 ---
 <div>
 	<h2>W3C WoT – Scripting API Task Force</h2>
-	<p>
-		Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
-	</p>
+    <p>
+		The WoT Scripting API task force is responsible for
+		specifying an application programming interface (API) representing
+		the WoT Interface that allows scripts to discover, operate Things
+		and to expose locally defined Things characterized by WoT Interactions.
+    </p>
+    <div>
+		<h3>Informative Deliverables</h3>
+        <p>The WoT Scripting API TF has its own informative deliverable.
+        </p>
+        <div>
+			<h4>Web of Things (WoT) Scripting API</h4>
+            <p>Scripting is an optional building block in WoT and it is typically used in gateways or browsers that are able to run a WoT Runtime and script management, providing a convenient way to extend WoT support to new types of endpoints and implement WoT applications such as Thing Directory.
```

Comment:
Please also include "to define the behavior of Things", e.g.

```
providing a convenient way to define the behavior of exposed Things, 
extend WoT support to new types of endpoints and implement WoT applications.
```
I would probably leave out Thing Directory as an example for WoT application, since it is a special one, and has a larger scope than the Scripting implementation. Scripting is _using_ that, though, and in some context it can also implement it.

Reply:
Makes perfect sense. Changed with https://github.com/w3c/wot-marketing/pull/102/commits/de0a1012a1ab907314a35b2761d3885fa4b52632